### ISSUE TITLE
Cannot add reactions to messages in forum threads

### ISSUE THREAD

**Issue by ApoY2k** [DV8FromTheWorld/JDA#2213]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [X] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

### Expected Behaviour

When adding reactions to forum thread messages, JDA should be able to add reactions to messages in form threads

### Code Example for Reproduction Steps

```java
suspend fun executeFavCommand(storage: Storage, event: SlashCommandInteractionEvent) {
    val interaction = event.reply("Fetching candidates...").await()
    val original = interaction.retrieveOriginal().await()
    original.addReaction("👍").await()
    original.addReaction("👎").await()
}
```

### Code for JDABuilder or DefaultShardManagerBuilder used

```java
val jda = JDABuilder.createDefault(env.authToken).build()
```

### Exception or Error

```yesyes
java.lang.NullPointerException: Cannot invoke "net.dv8tion.jda.api.entities.GuildChannel.getPermissionContainer()" because "channel" is null
at net.dv8tion.jda.internal.entities.mixin.channel.middleman.GuildChannelMixin.hasPermission(GuildChannelMixin.java:48)
at net.dv8tion.jda.internal.entities.mixin.channel.middleman.GuildChannelMixin.checkPermission(GuildChannelMixin.java:51)
at net.dv8tion.jda.internal.entities.mixin.channel.middleman.GuildMessageChannelMixin.checkCanAccessChannel(GuildMessageChannelMixin.java:107)
at net.dv8tion.jda.internal.entities.mixin.channel.middleman.MessageChannelMixin.addReactionBy
```

**Comment by MinnDevelopment** [comment]
Forums are implemented by #2184, can you try with that instead?

**Comment by ApoY2k** [comment]
@MinnDevelopment Will give it a try, thanks for the hint

**Comment by MinnDevelopment** [comment]
Fixed in [5.0.0-alpha.19](https://github.com/DV8FromTheWorld/JDA/releases/tag/v5.0.0-alpha.20)
